core/controllers/admin.py

```diff
@@ -752,22 +928,49 @@ class AddContributionRightsHandler(base.BaseHandler):
     """Handles adding contribution rights for contributor dashboard page."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'username': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            },
+            'category': {
+                'schema': {
+                    'type': 'basestring',
+                    'choices': [
+                        constants.CONTRIBUTION_RIGHT_CATEGORY_REVIEW_TRANSLATION, # pylint: disable=line-too-long
```

Comment:
Can you define the choices in constants.ts as `CONTRIBUTION_RIGHT_CATEGORIES` we can use it in frontend in future as well*?